cfme/web_ui/__init__.py

```diff
@@ -977,10 +977,11 @@ def table_in_object(table_title):
         table_title: Text in `p` element preceeding the table
     Returns: XPath locator for the desired table.
     """
+    # REVIEW: What uses this? I wasn't able to test it.
     # Description     paragraph with the text       following element    which is the table
     return (
         "//p[@class='legend' and normalize-space(text())='{}']"
-        "/following-sibling::*[1][@class='style3']").format(table_title)
+        "/following-sibling::table").format(table_title)
```

Comment:
https://github.com/RedHatQE/cfme_tests/commit/2dbf1093012ac29f6fb54469bd62fe0fb019b4d2#diff-7f7fa4e53ebd99803657de18407cebfeR150 There are a few lines in that file which reference it should give you a pretty good indication of where to find them.